quiche/src/pmtud.rs

```diff
@@ -1,92 +1,191 @@
 #[derive(Default)]
 pub struct Pmtud {
-    /// The current path MTU estimate.
-    cur_size: usize,
+    /// The current PMTU estimate.
+    current_mtu: usize,
 
-    /// The last MTU probe size that was attempted.
-    probe: usize,
+    /// The last PMTU probe size that was attempted.
+    probe_size: usize,
 
-    /// Indicated if Path MTU probe needs to be generated.
-    next_size: bool,
+    /// Whether or not a PMTU probe needs to be generated.
+    should_probe: bool,
 
-    /// Check config for PMTU variable.
-    enable: bool,
+    /// Whether or not PMTUD is enabled.
+    enabled: bool,
 }
 
 impl Pmtud {
     /// Creates new PMTUD instance.
-    pub fn new(cur_size: usize) -> Self {
+    pub fn new(initial_mtu: usize) -> Self {
         Self {
-            cur_size,
+            current_mtu: initial_mtu,
             ..Default::default()
         }
     }
 
-    /// Enables Path MTU Discovery for the connection.
+    /// Enables PMTUD for the connection.
     pub fn enable(&mut self, enable: bool) {
-        self.enable = enable;
+        self.enabled = enable;
     }
 
-    /// Returns enable status for Path MTU Discovery for the connection.
+    /// Whether or not PMTUD is enabled for the connection.
     pub fn is_enabled(&mut self) -> bool {
-        self.enable
+        self.enabled
     }
 
-    /// Specifies whether Path MTU Discovery should be performed at the next
-    /// opportunity, i.e., when the next packet is sent out if possible.
+    /// Specifies whether PMTUD should be performed at the next opportunity,
+    /// i.e., when the next packet is sent out if possible.
     ///
-    /// Once Path MTU has been discovered, this maybe set to false.
-    pub fn should_probe(&mut self, pmtu_next: bool) {
-        self.next_size = pmtu_next;
+    /// Once Path MTU has been discovered, this may be set to false.
+    pub fn set_should_probe(&mut self, should_probe: bool) {
+        self.should_probe = should_probe;
     }
 
     /// Returns the value of the Path MTU Discovery flag.
-    pub fn get_probe_status(&self) -> bool {
-        self.next_size
+    pub fn get_should_probe(&self) -> bool {
+        self.should_probe
     }
 
-    /// Sets the next Path MTU Discovery probe size.
-    pub fn set_probe_size(&mut self, pmtu_probe: usize) {
-        self.probe = pmtu_probe;
+    /// Sets the next PMTUD probe size.
+    pub fn set_probe_size(&mut self, probe_size: usize) {
+        self.probe_size = probe_size;
     }
 
-    /// Returns the next Path MTU Discovery probe size.
+    /// Returns the next PMTUD probe size.
     pub fn get_probe_size(&mut self) -> usize {
-        self.probe
+        self.probe_size
     }
 
-    /// Sets the current Path MTU Discovery size after a successful probe has
-    /// been performed.
-    pub fn set_current(&mut self, pmtu: usize) {
-        self.cur_size = pmtu;
+    /// Sets the current discovered PMTU after a successful probe has
+    /// been ACKed.
```

Comment:
nit: This is a tautology, the probe is only successful when it has been acked

The old comment seems more correct to me

Reply:
Yeah makes sense - restored the previous terminology